backend/src/main/java/de/lindatroesken/backend/model/MangelEntity.java

```diff
@@ -18,11 +17,17 @@
     @Column(name = "id", nullable = false)
     private Long id;
 
+    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @JoinColumn(name="user_id")
+    private UserEntity userEntity;
+
+
     @Column(name="description", columnDefinition="TEXT")
```

Comment:
columnDefinition="TEXT"

ist var 255 sonst zu kurz?

Reply:
Eventuell. Ich wollte den user input erstmal nicht in der Länge begrenzen.